### ISSUE TITLE
Support imgui-node-editor

### ISSUE THREAD

**Issue by GeTechG** [sonoro1234/LuaJIT-ImGui#37]
How about adding support for https://github.com/thedmd/imgui-node-editor
I would like to use this project in conjunction with my internal lua binding via cimgui, so not exactly the same project. I tried to do the generation steps myself and failed miserably.

**Comment by sonoro1234** [comment]
It is much more C++ than the all others ...

**Comment by GeTechG** [comment]
> It is much more C++ than the all others ...

Yes, I've already realized that by trying to understand the problem. But unfortunately all the others that you have already adapted have stopped development for one reason or another. I found one more https://github.com/Azzinoth/VisualNodeSystem but it was even more c++, and there are no worthy variants.

**Comment by sonoro1234** [comment]
https://github.com/Fattorino/ImNodeFlow  ?

**Comment by GeTechG** [comment]
> https://github.com/Fattorino/ImNodeFlow ?

Hmm, isn't it even more tied to c++, at a quick glance nodes are created by inheriting from BaseNode and as pins are created strictly. In general you can archive this issue, or leave it as is, as I did write a manual wrapper for my purposes https://github.com/thedmd/imgui-node-editor

**Comment by sonoro1234** [comment]
done with imgui-node-editor
